lib/resources/r4/location.yaml

```diff
@@ -0,0 +1,18 @@
+---
+name: Location
+field_name_base_url: http://hl7.org/fhir/R4/location-definitions.html#Location
+fields:
+
+  - name: physicalType
+    type: List of CodeableConcept
+    description: Physical form of the location, e.g. building, room, vehicle, road.
+    action: terminology
+    binding:
+      description: Physical form of the location.
+      terminology:
+        - display: Location physicalType
+          system: http://terminology.hl7.org/CodeSystem/location-physical-type
+          info_link: http://hl7.org/fhir/R4/valueset-location-physical-type.html
+        - display: Millennium Location Types
+          system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/222
+          info_link: http://localhost:3000/millennium/r4/proprietary-codes-and-systems/#code-set-222-location-type
```

Comment:
This is pointing to localhost?

Reply:
It should be: https://fhir.cerner.com/millennium/r4/proprietary-codes-and-systems/#code-set-222-location-type

Reply:
Fix for this:
https://github.com/cerner/fhir.cerner.com/commit/98602c8acc06a2175351fb1d3b11577736eec395